reconcile/test/cna/test_client.py

```diff
@@ -0,0 +1,45 @@
+from reconcile.cna.client import CNAClient
+
+
+def test_client_asset_type_metadata_init():
+    pass
+
+
+def test_client_list_assets_for_creator(mocker):
+    creator = "creator"
+    listed_assets = [
+        {
+            "asset_type": "null",
+            "id": "123",
+            "href": "url/123",
+            "status": "Running",
+            "creator": {"username": creator},
+        },
+        {
+            "asset_type": "null",
+            "id": "456",
+            "href": "url/456",
+            "status": "Running",
+            "creator": {},
+        },
+        {
+            "asset_type": "null",
+            "id": "789",
+            "href": "url/789",
+            "status": "Running",
+        },
+        {
+            "asset_type": "null",
+            "id": "000",
+            "href": "url/000",
+            "status": "Running",
+            "creator": {"username": "another_user"},
+        },
+    ]
+
+    mocker.patch.object(CNAClient, "list_assets", return_value=listed_assets)
```

Comment:
nit: reminder on `autospec` usage where it makes sense (there could be a good reason not to here).

Reply:
maybe covered in the other autospec related comment?